.github/workflows/lint.yml

```diff
@@ -7,7 +7,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-python@v5
         with:
-          python-version: '3.11'
+          python-version: '3.12'
```

Comment:
This will fail CI until corresponding genie_python dependency update PR is merged & a pypi release made.

Reply:
CI is still failing after genie python dependency update merged and release made

Reply:
Gave it a stern talking to in https://github.com/ISISComputingGroup/ibex_gui/pull/1783/commits/a474c108c04fb3502ad308821c677e441fdf4311